src/RouteJs/RouteJs.cs

```diff
@@ -88,5 +123,20 @@ private RouteInfo GetRoute(Route route)
 
 			return routeInfo;
 		}
+
+		// Example: "Posts/{postKey}/Edit" is converted to "posts/{postKey}/edit"
+		// Words inside parens are not matched.
+		private string GetUrl(string url)
+		{
+			if (!IsLowerCaseUrls)
+			{
+				return url;
+			}
+
+			return Regex.Replace(url, s_lowerCasePatternMatcher, (m) =>
+			{
+				return m.Value.ToLower();
```

Comment:
I wonder if it's worth making a separate interface for things that modify the URL (eg. `IUrlProcessor`) rather than directly embedding this here. Do you think there'll be many other instances of code that modifies the URL like this? If not, it's fine to keep this here.